xbmc/cores/DataCacheCore.h

```diff
@@ -119,8 +119,28 @@ class CDataCacheCore
   bool IsRenderClockSync();
 
   // player states
+  /*!
+   * @brief Notifies the cache core that a seek operation has finished
+   * @param offset - the seek offset
+  */
+  void SeekFinished(int64_t offset = 0);
```

Comment:
please try to avoid introducing new default arguments.. SeekFinished(0) is just more explicit.